tests/migrate/test_replace.py

```diff
@@ -148,6 +150,72 @@ def test_replace_last(self):
             ''')
         )
 
+    def test_replace_with_placeable(self):
+        msg = FTL.Message(
+            FTL.Identifier(u'hello'),
+            value=REPLACE(
+                'test.properties',
+                'hello',
+                {
+                    '#1': FTL.Placeable(
+                        EXTERNAL_ARGUMENT('user')
+                    )
```

Comment:
Would it work also with just `EXTERNAL_ARGUMENT('user')`?

Reply:
Yes, all other tests in this file are written that way :)

Reply:
It works because of https://github.com/projectfluent/python-fluent/pull/41/files/e731e8dfbd62cdc6762a5906473aff0925af1e7d#diff-f684603bb97b0b183acc54fecb78696fR176